pubspec.yaml

```diff
@@ -19,3 +19,9 @@ dev_dependencies:
   dart_dev: ^4.0.0
   dependency_validator: ^3.0.0
   test: ^1.18.2 
+
+dependency_overrides:
```

Comment:
TODO Once react releases, we'll need to remove this override and set the react dependency minimum to the new react release (line 12 above)